raw_data/preprocessed/africa/senegal_french.txt

```diff
@@ -0,0 +1,259 @@
+
+TITRE II - Prévention et lutte contre les pollutions et nuisances
+
+CHAPITRE I - Installations classées pour la protection de l'environnement
+
+(9) Sont soumis aux dispositions de la présente loi, les usines, ateliers, dépôts, chantiers, carrières et, d'une manière générale, les installations industrielles, artisanales ou commerciales exploitées ou détenues par toute personne physique ou morale, publique ou privée, et toutes autres activités qui présentent soit des dangers pour la santé, la sécurité, la salubrité publique, l'agriculture, la nature et l'environnement en général, soit des inconvénients pour la commodité du voisinage.
+
+(10) Les installations visées à l'article 9 sont divisées en deux classes. Suivant le danger ou la gravité des inconvénients que peut présenter leur exploitation, elles sont soumises soit à autorisation soit à déclaration.
+
+(11) La première classe comprend les installations qui présentent de graves dangers ou inconvénients pour les intérêts visés à l'article 9. L'exploitation de ces installations ne peut être autorisée qu'à condition que des mesures spécifiées, par arrêté ministériel, soient prises pour prévenir ces dangers ou inconvénients. La seconde classe comprend les installations qui, ne présentant pas d'inconvénients graves pour les intérêts visés à l'article 9, doivent respecter les prescriptions générales édictées par le Ministre chargé de l'environnement en vue d'assurer la protection de ces intérêts.
+
+(12) Les catégories d'installations soumises aux dispositions de la présente loi et le classement de chacune d'elles sont définis par arrêté du Ministre chargé de l'environnement, après avis des Ministères chargés de l'industrie et de la protection civile.
+
+(13) Les installations rangées dans la première classe doivent faire l'objet, avant leur construction ou leur mise en service, d'une autorisation d'exploitation délivrée par arrêté du Ministre chargé de l'environnement dans les conditions fixées par décret. Cette autorisation est obligatoirement subordonnée à leur éloignement, sur un rayon de 500 m au moins, des habitations, des immeubles habituellement occupés par des tiers, des établissements recevant du public et des zones destinées à l'habitation, d'un cours d'eau, d'un lac, d'une voie de communication, d'un captage d'eau. Les installations rangées dans la seconde classe doivent faire l'objet, avant leur construction ou leur mise en service, d'une déclaration adressée au Ministre chargé de l'environnement, qui leur délivre un récépissé dans les conditions fixées par décret. L'exploitant doit renouveler sa demande d'autorisation ou sa déclaration soit en cas de transfert, soit en cas d'extension, ou de modification notable des installations. En cas de mutation des droits d'exploitation, le nouvel exploitant est tenu de faire une déclaration adressée au Ministre chargé de l'environnement.
```

Comment:
Should the "500 m" be "500m"?

Reply:
According to [this guide](https://physics.nist.gov/cuu/Units/checklist.html#:~:text=There%20is%20a%20space%20between,superscript%20units%20for%20plane%20angle.&text=If%20the%20spelled%2Dout%20name,of%2035%2Dmillimeter%20film.%22), there should be a space between the two.